include/z64transition.h

```diff
@@ -23,6 +23,11 @@ typedef struct {
     /* 0xDC */ u16* zBuffer;
 } TransitionUnk; // size = 0xE0
 
+typedef enum {
+    /* 1 */ TRANS_INSTANCE_TYPE_END = 1,
+    /* 2 */ TRANS_INSTANCE_TYPE_START
```

Comment:
During discussion on the z_demo PR I (re)learned from @fig02 about `TRANS_MODE_FILL_IN` and `TRANS_MODE_FILL_OUT`, maybe this enum should use that terminology too

Reply:
Sure, it would probably be best if the terminology was the same.